common/dbconnector.cpp

```diff
@@ -232,7 +232,12 @@ SonicDBInfo& SonicDBConfig::getDbInfo(const std::string &dbName, const std::stri
         SWSS_LOG_ERROR("%s", msg.c_str());
         throw out_of_range(msg);
     }
-    auto& infos = foundNetns->second;
+    return foundNetns->second;
+}
+
+SonicDBInfo& SonicDBConfig::getDbInfo(const std::string &dbName, const std::string &netns)
+{
+    auto infos = getDbEntry(netns);
```

Comment:
```suggestion
    auto const& infos = getDbEntry(netns);
```
We want to make sure that we are taking a read-only reference.